packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -253,3 +250,13 @@ addGraphQLSchema(`
   }
 `)
 addGraphQLQuery('DigestPlannerData(digestId: String, startDate: Date, endDate: Date): [DigestPlannerPost]')
+
+createPaginatedResolver({
```

Comment:
Even though we're not using pagination here, I used a pagination resolver since:
 1) It's the easiest way I'm aware of connecting a frontend query to a SQL query.
 2) It comes with builtin caching which is particularly useful here.

Reply:
> It comes with builtin caching which is particularly useful here.

Have you measured the speed without the caching? Presumably it's fairly fast because it's just a single query but it would be good to double check.

I'm hesitant to use caching in places like this (i.e. implicitly and only used by one subcomponent) because it makes it a lot harder to catch and debug performance issues. Also if the page isn't used that often then users hit the uncached version very frequently (in this case if it gets ~1 user every 15 minutes there is a high chance of hitting the uncached version). I think if we ended up using this pattern a lot such that pages make several queries that each have their own independent cache then we would tend to get a lot of frustrating intermittent performance issues that would be hard to catch in development.

In this case seeing as it's likely to be fast (probably not the limiting factor for the page) without the cache I would much rather remove the caching

Reply:
This seems like a good pattern for doing "views that make complex db queries" though 👍